EXAMPLE/cluster.yml

```diff
@@ -3,22 +3,28 @@
 - name: Deploy the cluster
   hosts: localhost
   connection: local
-  roles:
-    - { role: clusterverse/clean, tags: [clusterverse_clean], when: clean is defined }
-    - { role: clusterverse/create, tags: [clusterverse_create] }
-    - { role: clusterverse/dynamic_inventory, tags: [clusterverse_dynamic_inventory] }
+  gather_facts: no
+  tasks:
+    - { name: "Get dependent roles via ansible-galaxy", local_action: "command ansible-galaxy install -r requirements.yml", tags: ["always"] }
```

Comment:
Shall we add `--force` after `command ansible-galaxy install -r requirements.yml` to make sure the correct clusterverse role (the one specified in requirements.yml) is imported every time the playbook is run?

Reply:
Yes, good idea.

Reply:
Added https://github.com/sky-uk/clusterverse/pull/70/commits/443f64c3442f658708315b9f7773db6eb7942fb5 to address